.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Parsers/DutchDateParserConfiguration.cs

```diff
@@ -162,7 +164,15 @@ public int GetSwiftMonthOrYear(string text)
         public bool IsCardinalLast(string text)
         {
             var trimmedText = text.Trim();
-            return trimmedText.Equals("last");
+            foreach (var s in lastCardinalTerms)
```

Comment:
Why not just 
`return lastCardinalTerms.Contains(trimmedText);`
?

Reply:
yes better, updated.